nix/all-engines.nix

```diff
@@ -114,4 +114,39 @@ in
       '';
     })
     { profile = "release"; };
+
+    packages.query-engine-wasm = lib.makeOverridable
+    ({ profile }: stdenv.mkDerivation {
+      name = "query-engine-wasm";
+      inherit src;
+      inherit (self'.packages.prisma-engines) buildInputs configurePhase dontStrip;
+
+      nativeBuildInputs = self'.packages.prisma-engines.nativeBuildInputs ++ (with pkgs; [wasm-pack wasm-bindgen-cli jq binaryen]);
+
+      buildPhase = ''
+      cd query-engine/query-engine-wasm
+      HOME=$(mktemp -dt wasm-pack-home-XXXX) WASM_BUILD_PROFILE=${profile} ./build.sh
```

Comment:
So, turns out wasm-pack can not work without `$HOME` and that was the reason for that permission error we were getting yesterday @aqrln.

Reply:
Nice, thanks for figuring this out!